README.rst

```diff
@@ -44,6 +44,26 @@ To add sphinx support::
 
     $ pip install rstcheck[sphinx]
 
+Pre-commit Hook with Extras
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To use ``rstcheck`` in a Git pre-commit hook with optional features:
+
+1. Install pre-commit::
+
+     pip install pre-commit
+
+2. Add to ``.pre-commit-config.yaml``::
+
+     - repo: https://github.com/rstcheck/rstcheck
+       rev: v6.2.4
```

Comment:
Please replace the version string with a placeholder like e.g. `<version>`.
This is to reduce the burden of the need to keep this part of the docs always up-to-date.